member-portal/django-dashboard-coreui/additional_references/README.md

```diff
@@ -118,8 +140,8 @@ $ sudo docker-compose pull && sudo docker-compose build && sudo docker-compose u
 docker-compose pull && docker-compose build && docker-compose up -d --remove-orphans
 ```
 
-Visit [http://localhost:5005](http://localhost:5005) in your browser. The app should be up & running.
-* For [login](http://localhost:5005/login/)
+Visit [http://localhost:8080](http://localhost:8080) in your browser. The app should be up & running.
+* For [login](http://localhost:8080/login/)
 
 > Stop and remove the app with docker-compose
 
```

Comment:
@peterbogdan one question, as we are using directly the .env file for the DEBUG in Django. Do we still need the section posted below ? *Line 176*:

* `~\member-portal\django-dashboard-coreui\django_app\gunicorn-cfg.py`

Reply:
@alanamircruz  not sure what the ask is
we still need the gunicorn-cfg.py to store the settings for gunicorn, the DEBUG from there is still retrieved from the .env file and used to "reload" gunicorn when it sees that files are changed